packages/@eventual/aws-cdk/src/entity-service.ts

```diff
@@ -136,6 +139,7 @@ export class EntityService<Service> {
       );
       this.configureReadWriteEntityTable(this.transactionWorker);
       props.workflowService.configureSendSignal(this.transactionWorker);
+      props.searchService?.configureSearch(this.transactionWorker);
```

Comment:
Why would a transaction need to call a search index?

Reply:
Good question. 

I've been thinking more about your idea for a transaction to be usable everywhere rather than making it a resource. You're right about that.